### PR TITLE
Arteria keys

### PR DESCRIPTION
**Pull request by johanherman**
Make sure arteria user can login everywhere
- add arteria_common role for all nodes (adds user + ssh keys)
- setup arteria user key in dev-dependencies
- use arteria user instead of vagrant on sisyphus/testuppmax

### CODE REVIEW

**Comment by johanherman** [comment]
But these things might be FUBAR now when the st2_system_user is changed.

**Comment by johanherman** [comment]
Should work now.

**Comment by johandahlberg** [comment]
Looks good!

File: ansible-st2-local/roles/arteria_core/defaults/main.yml [hunk 1]
```diff
@@ -1,11 +1,3 @@
...
-
-arteria_user: arteria
-arteria_uid: 4242
-arteria_group: arteria
-arteria_gid: 4242
-
-# Default to "prod". If set to "staging" some roles might have a slightly different
-# behavior, enabling debug mode and allowing extra integration tests capabilities.
-arteria_environment: prod
+arteria_disable_iptables: true
```
Comment (withrocks) [inline ansible-st2-local/roles/arteria_core/defaults/main.yml]: Wasn't this removed by @johandahlberg in an earlier commit? Should it go back in or is this a merge conflict issue?

Comment (johanherman) [inline ansible-st2-local/roles/arteria_core/defaults/main.yml]: Shouldn't be there it seems.

**Comment by withrocks** [comment]
Cool

**Pull request merged**